astropy/wcs/utils.py

```diff
@@ -14,13 +13,12 @@
            'proj_plane_pixel_area', 'is_proj_plane_distorted',
            'non_celestial_pixel_scales', 'skycoord_to_pixel',
            'pixel_to_skycoord', 'custom_wcs_to_frame_mappings',
-           'custom_frame_to_wcs_mappings', 'pixel_to_pixel']
+           'custom_frame_to_wcs_mappings', 'fit_wcs_from_points']
```

Comment:
This looks like a rebasing issue, ``pixel_to_pixel`` should not be removed from this list